scripts/servers_start.cmd

```diff
@@ -2,31 +2,58 @@
  
 call servers_stop.cmd
 
-call gradlew %1 build
-if ERRORLEVEL 1 goto end
+call gradlew build
+if ERRORLEVEL 1 goto error
 
+if "%1"=="-keepDB" goto updateDB
 echo.
 echo Recreating Yona database
 echo.
 call mysql --user=%YONA_DB_USER_NAME% --password=%YONA_DB_PASSWORD% < scripts\recreateYonaDB.sql
-if ERRORLEVEL 1 goto end
+if ERRORLEVEL 1 goto error
 
+:updateDB
 call gradlew :dbinit:liquibaseUpdate
-if ERRORLEVEL 1 goto end
+if ERRORLEVEL 1 goto error
 
+echo.
 echo Verifying the database schema
+echo.
 call gradlew :dbinit:bootRun
-if ERRORLEVEL 1 goto end
+if ERRORLEVEL 1 goto error
 
-start "Admin service" cmd /c gradlew %1 :adminservice:bootRun
-start "Analysis service" cmd /c gradlew %1 :analysisservice:bootRun
-start "App service" cmd /c gradlew %1 :appservice:bootRun
-start "Batch service" cmd /c gradlew %1 :batchservice:bootRun
+call gradlew adminservice:build && start "Admin service" java -Xdebug -Xrunjdwp:transport=dt_socket,address=8840,server=y,suspend=n -jar adminservice\build\libs\adminservice-0.0.8-SNAPSHOT-full.jar --server.port=8080 --management.port=9080 --spring.jpa.hibernate.ddl-auto=none
+call gradlew analysisservice:build && start "Analysis service" java -Xdebug -Xrunjdwp:transport=dt_socket,address=8841,server=y,suspend=n -jar analysisservice\build\libs\analysisservice-0.0.8-SNAPSHOT-full.jar --server.port=8081 --management.port=9081 --spring.jpa.hibernate.ddl-auto=none
+call gradlew appservice:build && start "App service" java -Xdebug -Xrunjdwp:transport=dt_socket,address=8842,server=y,suspend=n -jar appservice\build\libs\appservice-0.0.8-SNAPSHOT-full.jar --server.port=8082 --management.port=9082 --spring.jpa.hibernate.ddl-auto=none
+call gradlew batchservice:build && start "Batch service" java -Xdebug -Xrunjdwp:transport=dt_socket,address=8843,server=y,suspend=n -jar batchservice\build\libs\batchservice-0.0.8-SNAPSHOT-full.jar --server.port=8083 --management.port=9083 --spring.jpa.hibernate.ddl-auto=none
 
-set GRADLE_OPTS=
+echo.
 echo Wait until all services are started.
-pause
-curl -X PUT --header "Content-Type: application/json" -d @dbinit/data/activityCategories.json http://localhost:8080/activityCategories/
+echo.
+set CURLOPT=-s --retry-connrefused --retry-delay 3 --retry 20
+curl %CURLOPT% http://localhost:8080/activityCategories/ > nul
+if ERRORLEVEL 1 goto error
+curl %CURLOPT% http://localhost:8081/relevantSmoothwallCategories/ > nul
+if ERRORLEVEL 1 goto error
+curl %CURLOPT% http://localhost:8082/activityCategories/ > nul
+if ERRORLEVEL 1 goto error
+
+echo.
+echo Load the activity categories
+echo.
+curl -f -X PUT --header "Content-Type: application/json" -d @dbinit/data/activityCategories.json http://localhost:8080/activityCategories/
+if ERRORLEVEL 1 goto error
+
+echo.
+echo.
+echo Start the integration tests
```

Comment:
Can't you simply hit ctrl-c when the tests run?
The paus option means that you've to look after it and come back in the middle. In your case, I expect you will want to load the activity categories, so you can't break it at "pause", but need to make it do the next step, right?